### PR TITLE
[Feat] added country flags

### LINKED ISSUE #3511: Adding country flags in front of the name in dropdown in Settings Section
**Actual Behaviour**
No country flags in front of the name in the dropdown in Settings Section

**Expected Behaviour**
Adding country flags in front of the name in dropdown in Settings Section

**Steps to reproduce it**
1. Go to Settings
2. Select mobile option on sidebar
3. Click the dropdown

**Screenshots of the issue**
Screenshot 2020-02-22 15:42:59

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by BLasan**
Fixes #3511 

Changes: Added react-emoji-render. Please run the command npm install react-emoji-render --save 
                or yarn add react-emoji-render according to your package manager. I have added 
                package.json which consistes the dependency for that.

Demo Link: https://pr-3514-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot from 2020-04-05 10-12-01

### CODE REVIEW

**Comment by BLasan** [comment]
> looks like we don't have one in the project. both being ignored.

Have you already installed that dependency in your project? It's useful in displaying emojis

**Comment by snitin315** [comment]
> > It looks like we don't have one in the project. both being ignored.
> 
> Have you already installed that dependency in your project? It's useful in displaying emojis

I was talking about the lock files.

**Comment by BLasan** [comment]
> > > It looks like we don't have one in the project. both being ignored.
> > 
> > 
> > Have you already installed that dependency in your project? It's useful in displaying emojis
> 
> I was talking about the lock files.
> 
> https://github.com/fossasia/susi.ai/blob/46c936816320f68e20c142789fe8dd211248de2f/.gitignore#L24



> > > It looks like we don't have one in the project. both being ignored.
> > 
> > 
> > Have you already installed that dependency in your project? It's useful in displaying emojis
> 
> I was talking about the lock files.
> 
> https://github.com/fossasia/susi.ai/blob/46c936816320f68e20c142789fe8dd211248de2f/.gitignore#L24

Yes. It's not allowing to add package-lock.json. I force added that file. Any thing I have to do to fix that?

**Comment by snitin315** [comment]
if that's the case it shouldn't be committed forcefully.
/cc @akshatnitd

**Comment by BLasan** [comment]
> if that's the case it shouldn't be committed forcefully.
> /cc @akshatnitd

Can I remove that and push again ?

**Comment by snitin315** [comment]
> > if that's the case it shouldn't be committed forcefully.
> > /cc @akshatnitd
> 
> Can I remove that and push again ?

Remove it.

File: src/components/Settings/MobileTab.react.js [hunk 1]
```diff
@@ -145,6 +146,7 @@ class MobileTab extends React.Component {
               {countryData.countries.all[i].name +
                 ' ' +
                 countryData.countries.all[i].countryCallingCodes[0]}
+              <Emoji text={countryData.countries.all[i].emoji} />
             </MenuItem>
```
Comment (snitin315) [inline src/components/Settings/MobileTab.react.js:150]: how about ~~`country CallingCode flag`~~ ` flag CallingCode country`

Comment (BLasan) [reply src/components/Settings/MobileTab.react.js:150]: That's better. I'll fix it

**Pull request merged**